pkg/cli.go

```diff
@@ -52,63 +56,70 @@ func NewApp() *cli.App {
 					},
 				},
 			},
```

Comment:
The whole create function is not needed.

Reply:
cool

Reply:
done